src/commands/clean-columns.ts

```diff
@@ -1,60 +0,0 @@
-import fs = require('fs');
-import path = require('path');
-import request = require('request');
-
-type Result = {
-    data: {
-        repository: {
-            project: {
-                columns: {
-                    edges: Array<{
-                        node: {
-                            cards: {
-                                edges: Array<{
-                                    node: {
-                                        databaseId: number;
-                                        content: {
-                                            state: "OPEN" | "MERGED" | "CLOSED";
-                                            number: number;
-                                        };
-                                    };
-                                }>;
-                            };
-                        };
-                    }>;
-                };
-            };
-        };
-    };
-};
-
-const headers = {
-    "User-Agent": "RyanCavanaugh ColumnCleaner",
-    "Authorization": `token ${process.env['AUTH_TOKEN']}`,
-    "Accept": "application/vnd.github.inertia-preview+json"
-}
-
-
-let cleanCount = 0;
-fs.readFile(path.join(__dirname, '../clean-columns-query.graphql'), { encoding: 'utf-8' }, (err, query) => {

```

Comment:
Nice! I think this is the only reference to https://github.com/DefinitelyTyped/dt-mergebot/blob/master/clean-columns-query.graphql so we can add that file to this PR?

Reply:
Bah, you're right -- I wanted to check that one but forgot...